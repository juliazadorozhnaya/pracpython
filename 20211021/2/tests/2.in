:sin x y 2*(x+y)
sin 1 2
:const 1000
const
:cos x y z u tan(x+y+z+u)
cos 1 2 3 4
quit "fun:{}strokaz:{}"