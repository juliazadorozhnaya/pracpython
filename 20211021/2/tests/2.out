6
1000
0.6483608274590866
"fun:4strokaz:9"
